src/countdown/countdown.d.ts

```diff
@@ -0,0 +1,15 @@
+declare namespace Countdown {
+  interface CountdownProps {
+    startDate: Date;
+    testid?: string;
+    countDownIntervalInSeconds?: number;
+    alwaysShowSeconds?: boolean;
+    onEnd?: () => void;
+    customClassName?: string;
+  }
+
+  interface CountdownItem {
+    id: "days" | "hours" | "minutes" | "seconds";
+    count: string | number;
+  }
+}
```

Comment:
Why are we using a namespace here ? Can't we just do the following ?
```suggestion
  interface CountdownProps {
    startDate: Date;
    testid?: string;
    countDownIntervalInSeconds?: number;
    alwaysShowSeconds?: boolean;
    onEnd?: () => void;
    customClassName?: string;
  }

  interface CountdownItem {
    id: "days" | "hours" | "minutes" | "seconds";
    count: string | number;
  }
```